File: parser.js
const fs = require('fs');
const pdf = require('pdf-parse');

async function extract() {
    let data = await pdf(fs.readFileSync('./FaizaHost_PRD (1).pdf'));
    console.log("--- PRD ---");
    console.log(data.text);

    data = await pdf(fs.readFileSync('./Overview.pdf'));
    console.log("--- OVERVIEW ---");
    console.log(data.text);
}
extract().catch(e => console.error(e));

File: parser.py
import os
from PyPDF2 import PdfReader

try:
    print("--- PRD ---")
    reader = PdfReader("FaizaHost_PRD (1).pdf")
    for page in reader.pages:
        print(page.extract_text())
        
    print("--- OVERVIEW ---")
    reader = PdfReader("Overview.pdf")
    for page in reader.pages:
        print(page.extract_text())
except Exception as e:
    print(f"Error: {e}")
